src/modules/appeals-service/AppealsService.ts

```diff
@@ -1,45 +1,55 @@
-import axios, { AxiosError, AxiosRequestConfig, AxiosResponse } from 'axios';
+import axios, { AxiosError, AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios';
 import { CREATED, NOT_FOUND, UNAUTHORIZED, UNPROCESSABLE_ENTITY } from 'http-status-codes';
-import { AppealNotFoundError, AppealServiceError, AppealUnauthorisedError, AppealUnprocessableEntityError } from './errors';
+import { AppealNotFoundError, AppealServiceError, AppealUnprocessableEntityError } from './errors';
 
 import { loggerInstance } from 'app/middleware/Logger';
 import { Appeal } from 'app/models/Appeal';
+import { RefreshTokenService } from 'app/modules/refresh-token-service/RefreshTokenService';
 
 export class AppealsService {
 
-    constructor(private readonly uri: string) {
+    private readonly axiosInstance: AxiosInstance;
+
+    constructor(private readonly uri: string, private readonly refreshTokenService: RefreshTokenService) {
         this.uri = uri;
+        this.refreshTokenService = refreshTokenService;
+        this.axiosInstance = axios.create();
     }
 
-    public async getAppeal(companyNumber: string, appealId: string, token: string): Promise<Appeal> {
+    public async getAppeal(companyNumber: string, appealId: string, accessToken: string,
+                           refreshToken: string): Promise<Appeal> {
 
         this.checkArgumentOrThrow(companyNumber, 'Company number is missing');
         this.checkArgumentOrThrow(appealId, 'Appeal id is missing');
-        this.checkArgumentOrThrow(token, 'Token is missing');
+        this.checkArgumentOrThrow(accessToken, 'Token is missing');
+
```

Comment:
should we be checking for refresh token here aswell?: 
```suggestion
        this.checkArgumentOrThrow(refreshToken, 'Refresh token is missing');
```